### PR TITLE
feat: Certificate completion page

### PR DESCRIPTION
**Pull request by chamod-gamage**
## Notion ticket link

[Certificates: End of Module Page](https://www.notion.so/uwblueprintexecs/Certificates-End-of-Module-Page-f378628b944e45eaaeee37937d223a3e)

## Implementation description
image

* Implements the `certificate/[toolId]` page, uses dynamic data
* Hooks up the following buttons/links
  * Toolkit > Tool breadcrumb
  * Exit and Save Progress
  * Download
  * Back to Toolkit
  * Remaining button: Share
* Adds top bar (with HUG logo linked to home) to Module page and this certificate page
* Final slide ('next' button forced to 'finish') links to Certificate page

https://www.loom.com/share/e26f328b6b374d228b14f5c44ee2c705

## Steps to test
2. Visit http://localhost:3000/
3. Ensure you are logged in, if not log in at http://localhost:3000/login
4. Ensure a tool with a module exists, if not create one at http://localhost:3000/admin/dashboard/modules
5. Click into a module and go to final slide
6. Click Finish, check if the certificate page pops up with correct info
7. Test the buttons listed above, see if they work as intended

## What should reviewers focus on?
* Make sure the page is styled correctly and functional in terms of its buttons

## Checklist
- [ ] My PR name is descriptive and in imperative tense
- [ ] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [ ] I have run the appropriate linter(s)
- [ ] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: pages/certificate/[tool].tsx [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+        );
+    };
+    const handleBack = () => {
+        router.push("/");
+    };
+
+    useEffect(() => {
+        const getCurrentUser = async () => {
+            const currentUser = await Auth.currentAuthenticatedUser();
+            setUser(currentUser);
```
Comment (mkanoria) [inline pages/certificate/[tool].tsx:47]: Wonder if `setUser(await Auth.currentAuthenticatedUser())` would work? Not a big difference tho haha

**Review by mkanoria — APPROVED** [review]
Looks great, the demo is amazing!

File: pages/certificate/[tool].tsx [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+                                alignItems={large ? "flex-start" : "center"}
+                            >
+                                <Flex
+                                    flexDirection={large ? "row" : "column"}
+                                    justifyContent={
+                                        large ? "flex-start" : "center"
+                                    }
+                                    mx={"-10px"}
+                                >
+                                    <Button
```
Comment (tawnyzhao) [inline pages/certificate/[tool].tsx:118]: Setting this button to loading while the data is fetching will avoid a default pdf from being generated

**Review by tawnyzhao — CHANGES_REQUESTED** [review]

**Comment by chamod-gamage** [comment]
Note that the Share button has been hidden in the recent commits.

**Pull request merged**